src/formatDate.js

```diff
@@ -51,6 +51,49 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const oldDivider = fromFormat[3];
+  const newDivider = toFormat[3];
+  const oldDate = date.split(oldDivider);
+  const oldDateFormat = fromFormat.slice(0, -1);
+  const newDateFormat = toFormat.slice(0, -1);
+  const oldDateObject = { normalize: toNormalizeDate };
+  const newDate = [];
+
+  for (let i = 0; i < oldDate.length; i++) {
+    oldDateObject[oldDateFormat[i]] = oldDate[i];
+  }
+
+  oldDateObject.normalize(newDateFormat);
```

Comment:
what's the reason of adding this function as a method for new object? let's create universal function, that will accept 2 objects (old date and new date), and make needed logic